sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumerClient/EventHubConsumerClientTests.cs

```diff
@@ -452,7 +455,17 @@ public async Task CloseAsyncSurfacesExceptionsForActiveTransportConsumers()
                 .Setup(consumer => consumer.CloseAsync(It.IsAny<CancellationToken>()))
                 .Returns(Task.FromException(new InvalidCastException()));
 
-            try { await using var receiver = consumer.CreatePartitionReceiver("0", EventPosition.Earliest); } catch {}
+
+
+
+            try
```

Comment:
Omg. So many blank lines.

Reply:
um....  yeah.   I've got nothing there.   I'll fold that fix into the next PR so that I can close this out.